source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -222,6 +223,15 @@ def get_address(self, obj):
         return AddressSerializer(obj).data
 
 
+class LineSerializer(NestedDictGenericField):
```

Comment:
OK, so you need a DictSerializer and a PbSerializer.
Can we name them differently to avoid mixup?
2 ideas :
* PbLineSerializer and DictLineSerializer (both in pt.py) if we want them to match (probably with a comment to "enforce" that match)... With `display_none=True` for code and so on.
* or LineSerializer and GeocodeLineSerializer if we don't want them to match precisely (but then we should probably document it as @kinnou02 said)

Reply:
the fact that it is nested in geocode_json is not enough for you ?

the point is the same for all the types of this module (`StopAreaSerializer`, `AddressSerializer`, ...)

Reply:
to be a bit more precise, the serializer is very tightly coupled to mimir's response, it is not a `DictLineSerializer`, it is a `MimirLineSerializer`, that's why I think it should be kept in `geocode_json` (and thus I find the naming fine, but I can change it it you prefer)

Reply:
OK, I didn't check the other names.
It's probably more some tests to add to avoid diffs between existing autocomplete services, then.
But it's for another PR if any :+1: